Index: include/cmdln_macro.h
===================================================================
--- include/cmdln_macro.h	(revision 12908)
+++ include/cmdln_macro.h	(working copy)
@@ -238,6 +238,11 @@
       ARG_STRING,									\
       NULL,										\
       "Word trigram language model input file" },					\
+{ "-lm_trie",											\
+          ARG_BOOLEAN,									\
+          "no",											\
+          "Whether to use trie structure to hold LM"	\
+},														\
 { "-lmctl",										\
       ARG_STRING,									\
       NULL,										\
